pom.xml

```
@@ -37,7 +37,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <awssdk.version>2.13.18</awssdk.version>
+        <awssdk.version>2.15.0</awssdk.version>
```

@akshay-aws
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/pull/267#issuecomment-632796555

@jotompki should we go with the latest available version instead? Also, we should have a process to keep this updated
```suggestion
        <awssdk.version>2.15.17</awssdk.version>
```